docs/cli/use.rst

```diff
@@ -76,8 +76,11 @@ By default, some summary tables are database `views <https://www.postgresql.org/
 
     python ocdskingfisher-views-cli add-view 123 "The note" --name the_name --tables-only
 
+This option is useful if you want to create a schema that does not rely on data from the Kingfisher Process database.  This means that if you give users permissions to this schema only then they will have a useful subset of the data, without them needing to have permissions to all data within Kingfisher.
```

Comment:
Is the only other use case to generate the ERD? Does the lack of foreign key relationships when using database views have any significant consequence from user's perspective?

Reply:
As someone analysing the data it is useful to do `\d <table>` and quickly see the foreign key constraints to know where to join to.  Otherwise you will have to do `\d+ <view_name>` look at the view definition and then do `\d <table>` on the table in the statement.  So it has marginal benefit, but I was not sure it was big enough to document?